main.go

```diff
@@ -29,7 +29,7 @@ func setupFlags(app *kingpin.Application) {
 	app.Flag("config", "Load JSON configuration from file.").Action(loadConfig).String()
 	app.Flag("disable", "Disable previously enabled linters.").PlaceHolder("LINTER").Short('D').Action(disableAction).Strings()
 	app.Flag("enable", "Enable previously disabled linters.").PlaceHolder("LINTER").Short('E').Action(enableAction).Strings()
-	app.Flag("linter", "Define a linter.").PlaceHolder("NAME:COMMAND:PATTERN").StringMapVar(&config.Linters)
+	app.Flag("linter", "Define a linter.").PlaceHolder("NAME:COMMAND:PATTERN").Action(cliLinterOverrides).StringMap()
```

Comment:
I'm not that familiar with `kingpin`. Is the `StringMap()` necessary here? It seems like it returns a value that we aren't using. Maybe it does something else as well?

Reply:
Yes, that is exactly correct. `StringMap()` just ensures the input is correctly formatted and typed.